module/.golangci.yml

```diff
@@ -1,16 +1,16 @@
 linters:
   enable:
-    # - bodyclose
+      - bodyclose
       - deadcode
-    # - depguard
-    # - dogsled
-    # - dupl
+      - depguard
+      - dogsled
+      - dupl
     # - errcheck
     # - exhaustivestruct
```

Comment:
looks like we've whittled this down to just godox, gofmt, goimports remaining, with errorchk and exhaustivestruct being two that require more strategy to enable.

Good work.